gpflow/kernels.py

```diff
@@ -683,42 +683,42 @@ def Kdiag(self, X, presliced=False):
 
 class Periodic(Kernel):
     """
-    The periodic kernel. Defined in  Equation (47) of
+    The periodic family of kernels. Can be used to wrap any Stationary kernel
+    to transform it into a periodic version. The canonical form (based on the
+    RBF kernel) can be found in Equation (47) of
 
     D.J.C.MacKay. Introduction to Gaussian processes. In C.M.Bishop, editor,
     Neural Networks and Machine Learning, pages 133--165. Springer, 1998.
 
-    Derived using an RBF kernel once mapped the original inputs through
-    the mapping u=(cos(x), sin(x)).
+    The derivation can be achieved by mapping the original inputs through the
+    transformation u = (cos(x), sin(x)).
 
-    The resulting kernel can be expressed as:
-        k(r) =  σ² exp{ -0.5 sin²(π r / γ) / ℓ²}
+    The resulting RBF-based kernel can be expressed as:
+        k(r) = σ² exp{ -0.5 sin²(π r / γ) / ℓ² }
 
     where:
-    r  is the Euclidean distance between the input points
+    r is the Euclidean distance between the input points
     ℓ is the lengthscale parameter,
     σ² is the variance parameter,
     γ is the period parameter.
 
     (note that usually we have a factor of 4 instead of 0.5 in front but this is absorbed into lengthscale
     hyperparameter).
     """
-
-    def __init__(self, input_dim, period=1.0, variance=1.0,
-                 lengthscales=1.0, active_dims=None, name=None):
-        # No ARD support for lengthscale or period yet
```

Comment:
Is this comment still applicable - it still sets `self.ARD = False` below? Should this just be a pass-through property that returns `self.base.ARD`?

Reply:
I guess the comment is still applicable in that there is still no ARD support for period.

However, I will update self.ARD to be a pass-through and maybe in a follow-up PR we can add support for different periods per dimension...in which case we might want to rename period -> periods?